packages/@aws-cdk/aws-codepipeline/lib/stage.ts

```diff
@@ -42,17 +42,21 @@ export class Stage implements IStage {
   /**
    * Get a duplicate of this stage's list of actions.
    */
-  public get actions(): FullActionDescriptor[] {
+  public get actionDescriptors(): FullActionDescriptor[] {
```

Comment:
breaking change? Why is this even in the public API?

Reply:
It's not a breaking change - the `Stage` class is private to the `@aws-cdk/aws-codepipeline` package.